lib/fastly/client.rb

```diff
@@ -13,6 +13,7 @@ def initialize(opts)
       @user     = opts.fetch(:user, nil)
       @password = opts.fetch(:password, nil)
       @customer = opts.fetch(:customer, nil)
+      @oldpurge = opts.fetch(:use_old_purge_method, false)  
```

Comment:
> IIRC we had some people in networks which didn't allow certain HTTP methods. I could be misremembering.

That's correct. It's a minor edge case, but not every host allows nonstandard methods.